ldms/src/sampler/spank/Plugin_slurm_notifier.man

```diff
@@ -31,6 +31,10 @@ specifies the name of publishing stream. The default value is \fIslurm\fR.
 is the number of seconds determining the time-out of the LDMS connections
 (default \fI5\fR).
 
+.BI debug=1
+enables certain plugin management debugging messages to the user's slurm log.
```

Comment:
I'm not sure this is true, if enabled, messages go to the user's job output, not the 'slurm log'. I think this is just a wording issue.